src/ServerPrivate.cc

```diff
@@ -436,6 +449,42 @@ bool ServerPrivate::WorldsService(ignition::msgs::StringMsg_V &_res)
   return true;
 }
 
+//////////////////////////////////////////////////
+bool ServerPrivate::ServerControlService(
+  const ignition::msgs::ServerControl &_req, msgs::Boolean &_res)
+{
+  _res.set_data(false);
+
+  if (_req.stop())
+  {
+    this->Stop();
+    _res.set_data(true);
+  }
+
+  // TODO(anyone): implement world cloning
```

Comment:
We're trying to get rid of the `anyone`s (see the [style guide](https://google.github.io/styleguide/cppguide.html#TODO_Comments)). You can put on your name, or if you'd rather not, you can put `chapulina`.

Reply:
Okay, I definitely do not feel like adding support for these functions, so I gave these TODOs to you :)